CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- `new-home` route is now `home`
+- `home` route is now `old-home`
```

Comment:
There's no waffle flag for switching back to `old-home`, right? If not, seems better to delete it. If we want the old homepage back, it's always here in git.